backend/prisma/migrations/20240628172722_init_user_and_trans/migration.sql

```diff
@@ -0,0 +1,47 @@
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "email" TEXT NOT NULL,
+    "password" TEXT NOT NULL,
+    "city" TEXT NOT NULL,
+    "salary" TEXT NOT NULL,
+    "roommates" INTEGER NOT NULL,
+    "children" INTEGER NOT NULL,
+    "job" TEXT NOT NULL,
+
+    CONSTRAINT "User_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "Transactions" (
+    "id" SERIAL NOT NULL,
+    "userId" INTEGER NOT NULL,
+    "rent" DOUBLE PRECISION NOT NULL,
```

Comment:
One thing to think about is how to handle money in code. This is a commonly talked about issue in programming (google search will reveal many discussions). Float may not be the best way to represent money because of floating point math issues. Is there another way you could more reliably represent money?